lib/sinon/util/fake_server.js

```diff
@@ -80,12 +81,14 @@ var fakeServer = {
 
         return server;
     },
+
     configure: function (config) {
         var whitelist = {
```

Comment:
If I understand the original authors intent correctly The whitelist object is being used as a [Set](https://en.wikipedia.org/wiki/Set_%28abstract_data_type%29) (so the values are un-important).  As well as looking a little cleaner, it also allows O(1) lookup of the values (not that it would make any meaningful difference in this particular situation ;)